shared/ui-components/styles/forms.scss

```diff
@@ -8,7 +8,7 @@
   label,
   .label {
     @apply text-gray-750;
-    font-size: 0.95rem;
+    font-size: .875rem;
```

Comment:
I hesitate to sign off on a font size change to all labels at this point. Can you check with @slowbot to confirm this is what we want?